Index: src/MyFirstJavaClass.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import java.util.Scanner;\r\n\r\npublic class MyFirstJavaClass {\r\n    public static void main(String[] args) {\r\n        Scanner scanner = new Scanner(System.in);\r\n        int num = scanner.nextInt();\r\n\r\n        if (num == 100){\r\n            for (int i = num; i > 0; i--) {\r\n                if (num % i == 0) {\r\n                    System.out.println(i);\r\n                }\r\n            }\r\n        }\r\n\r\n        if (true){\r\n            System.out.println(\"Hello git\");\r\n        }\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/MyFirstJavaClass.java b/src/MyFirstJavaClass.java
--- a/src/MyFirstJavaClass.java	(revision 19d2da62ec916a19d076057c5a4f74275dd5a0b7)
+++ b/src/MyFirstJavaClass.java	(date 1696529087232)
@@ -16,5 +16,10 @@
         if (true){
             System.out.println("Hello git");
         }
+
+        //как переключаться
+        //ветка как переключаться
+        // merge rebase
+        //повторить
     }
 }
